app/notifications/receive_notifications.py

```diff
@@ -128,6 +127,13 @@ def fetch_potential_services(inbound_number, provider_name):
         ))
         statsd_client.incr('inbound.{}.failed'.format(provider_name))
         return False
+
+    str_permissions = [p.permission for p in potential_services[0].permissions]
+    if set([INBOUND_SMS_TYPE, SMS_TYPE]).issubset(set(str_permissions)) is False:
+        current_app.logger.error(
```

Comment:
This code isn't used anywhere else in this way at the moment, so not sure if work refactoring?